### PR TITLE
ENH: move broker builders to utils so we don't need run engine to use…

### PR DESCRIPTION
**Pull request by CJ-Wright**
… them

### CODE REVIEW

File: databroker/tests/conftest.py [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-import shutil
-import os
 import sys
-import tempfile
-import tzlocal
-from databroker import Broker
+
 import pytest
 
+from databroker.tests.utils import build_sqlite_backed_broker, \
```
Comment (tacaswell) [inline databroker/tests/conftest.py]: `()` not `\` please.

File: databroker/tests/conftest.py [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
+    build_pymongo_backed_broker
 
 if sys.version_info >= (3, 0):
-    from bluesky.tests.conftest import fresh_RE as RE
```
Comment (tacaswell) [inline databroker/tests/conftest.py]: If you remove this line, you don't need to do the rest.

Comment (CJ-Wright) [inline databroker/tests/conftest.py]: Ok, I thought a bunch of your tests depended on RE.

**Comment by CJ-Wright** [comment]
@tacaswell Ready for more review I think!

**Comment by tacaswell** [comment]
I don't really understand _why_ you don't want a RE, but sure.

**Comment by CJ-Wright** [comment]
One could have a databroker without a RE, eg back-ending an analysis pipeline with the databroker.

**Comment by tacaswell** [comment]
fair

**Comment by tacaswell** [comment]
I just really like the RunEngine ....

**Comment by CJ-Wright** [comment]
Don't worry, I like it too.

**Pull request merged**